index.html

```diff
@@ -181,6 +181,71 @@ <h1>SKILLS</h1>
       </div>
     </div>
   </section>
+
+  <!-- Form section -->
+  <section class="section-form">
+    <div class="form-paragraph">
+      <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    </div>
+    <form action="https://formspree.io/f/mrgjjrrq" method="post">
+      <input name="Name" id="name" type="text" maxlength="30" minlength="5" required placeholder="Full name"
+        class="input-field">
```

Comment:
- [ ] [Optional] Kindly remove `minlength` attribute because it's not required to do and there are names existing with 3 or 4 letters.